Source/LoadByOS/LinuxConfigApp/LinuxInterop.cs

```diff
@@ -25,21 +25,59 @@ public LinuxInterop(params object[] values) { }
 		public void SetFolderIcon(string image, string directory) => throw new PlatformNotSupportedException();
 		public void DeleteFolderIcon(string directory) => throw new PlatformNotSupportedException();
 
-		public string ReleaseIdString => LibationScaffolding.ReleaseIdentifier.ToString() + (File.Exists("/bin/yum") ? "_RPM" : "");
+		public string ReleaseIdString => LibationScaffolding.ReleaseIdentifier.ToString() + (File.Exists("/bin/apt") ? "_DEB" : "_RPM");
```

Comment:
Are you aware that `apt` can't be used for RPM packages? I might have just misunderstood the code, but want to confirm. It solely supports what DPKG does – Deb packages.

Reply:
Yes.
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/conditional-operator